### DISCUSSION TITLE
Best practices to run GCP checks on all projects

### DISCUSSION THREAD

**Discussion by alon-u3d** [prowler-cloud/prowler#2367] [Q&A] [answered]
Hi,

I wanted to know what's the best practices to run GCP checks on all projects in an organization.

Thanks :)

**Comment by alon-u3d** [comment]
I first tried to list projects and iterate on all using gcloud config set $project_id but it seems like prowler is using the "project_id" from the service account credentials json.
Can you approve that the only way to run Prowler on multiple projects is to create/modify the credentials file per project?

**Comment by MrCloudSec** [comment, marked as answer]
Good point! 
This PR add the flag `--project-ids` and scans all the projects by default: https://github.com/prowler-cloud/prowler/pull/2393
In the next version 3.6.0, this feature will be available.
